modules/core/src/lib/attribute-transition-manager.js

```diff
@@ -173,27 +173,25 @@ export default class AttributeTransitionManager {
   // get current values of an attribute, clipped/padded to the size of the new buffer
   _getNextTransitionStates(transition, settings) {
     const {attribute} = transition;
-    const {size, offset, normalized} = attribute;
+    const {size, normalized} = attribute;
+    const multiplier = attribute.doublePrecision ? 2 : 1;
 
     let toState;
     if (attribute.constant) {
-      toState = new BaseAttribute(this.gl, {constant: true, value: attribute.value, size, offset});
+      toState = new BaseAttribute(this.gl, {constant: true, value: attribute.value, size});
     } else {
       toState = new BaseAttribute(this.gl, {
         constant: false,
         buffer: attribute.getBuffer(),
         divisor: 0,
         size,
-        offset,
```

Comment:
Why are we getting rid of `offset`? If the attribute we're transitioning has an `offset`, won't the transform feedback pass run with the wrong values (the values before the offset)?

Reply:
All we need is to run the transform with the same offset in from, to and feedback buffers. Always using 0 makes things easier.

Reply:
Hmmm - I suppose this is only true because we happen to just be using `mix` in this particular shader. If we were to use `length()` (as we would have to in order to support a `maxSpeed` setting for springs), for example, this wouldn't work, right?

My worry is that it reads like an oversight and leads the reader to believe `offset` and `stride` don't matter for transitions, when they just happen to not matter because of the specific calculations we're doing in this transition. If we really want to leave them out, maybe we leave a comment here about why we're intentionally doing that?

Reply:
For attributes that are packed by the `AttributeManager`, we do not have a situation where these two matter.

For externally provided interleaved buffers, because we did not pass `stride` before, transition never worked. In fact, I do not believe interleaved buffers work at all with any attribute that contains shader attributes (i.e. positions and transform matrices), given that we are overriding `offset` and `stride` internally. @tsherif